sql/driver/driver_test.go

```diff
@@ -0,0 +1,188 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Peter Mattis (peter@cockroachlabs.com)
+
+package driver
+
+import (
+	"database/sql"
+	"reflect"
+	"regexp"
+	"testing"
+
+	"github.com/cockroachdb/cockroach/server"
+	"github.com/cockroachdb/cockroach/util/leaktest"
+)
+
+func setup(t *testing.T) (*server.TestServer, *sql.DB) {
+	s := server.StartTestServer(nil)
+	db, err := sql.Open("cockroach", "https://root@"+s.ServingAddr()+"?certs=test_certs")
+	if err != nil {
+		t.Fatal(err)
+	}
+	return s, db
+}
+
+func cleanup(s *server.TestServer, db *sql.DB) {
+	_ = db.Close()
+	s.Stop()
+}
+
+func isError(err error, re string) bool {
```

Comment:
It is done in `client/table_test.go` which will be going away shortly. Strange that `util/testing.go` is compiled into the `util` package. That doesn't seem right.